File: Logica_Programacao/Atividade_Logica/numero_7.py
total = int(input('Total de Eleitores: '))
votobranco = int(input('Informe os votos em branco: '))
nulos = int(input('Informe votos nulos: '))
validos = int(input('Informe votos validos: '))

porBranco = (100*votobranco)/total
porNulo = (100*nulos)/total
porValidos = (100*validos)/total

print('Votos Brancos: {:.2f}'.format(porBranco),'%')
print('Votos Nulos: {:.2f}'.format(porNulo),'%')
print('Votos Validos: {:.2f}'.format(porValidos),'%')

File: Logica_Programacao/Atividade_Logica/numero_7.cpp
/* (FACCAT) Escreva um algoritmo para ler o número total de eleitores de um município, o
número de votos brancos, nulos e válidos. Calcular e escrever o percentual que cada um
representa em relação ao total de eleitores
*/

#include<iostream>
using namespace std;

int main(){
int total, validos, nulos, branco;
float percBranco, percNulos, percValidos;

// Solicita ao usuario os dados
cout<<"Total de eleitores: ";
cin>> total;
cout<<"Votos em branco: ";
cin>> branco;
cout<<"Votos nulos: ";
cin>> nulos;
cout<<"Votos validos: ";
cin>>validos;

// Caculos de percentual
percBranco = (100*branco)/total;
percNulos = (100*nulos)/total;
percValidos = (100*validos)/total;

// Exibe os resultados
cout<<"Percentual de votos branco..: "<<percBranco<<"%"<<endl;
cout<<"Percentual de votos nulos...: "<<percNulos<<"%"<<endl;
cout<<"Percntutal de votos validos.: "<<percValidos<<"%"<<endl;

return 0;
}
